nisystemlink/clients/feeds/_feeds_client.py

```diff
@@ -0,0 +1,171 @@
+"""Implementation of SystemLink Feeds Client."""
+
+from typing import BinaryIO, List, Optional
+
+from nisystemlink.clients import core
+from nisystemlink.clients.core._uplink._base_client import BaseClient
+from nisystemlink.clients.core._uplink._methods import delete, get, post
+from uplink import Part, Path, Query
+
+from . import models
+
+
+class FeedsClient(BaseClient):
+    """A set of methods to access the APIs of SystemLink Feeds Client."""
+
+    def __init__(self, configuration: Optional[core.HttpConfiguration] = None):
+        """Initialize an instance.
+
+        Args:
+            configuration: Defines the web server to connect to and information about
+                how to connect. If not provided, the
+                :class:`HttpConfigurationManager <nisystemlink.clients.core.HttpConfigurationManager>`
+                is used to obtain the configuration.
+
+        Raises:
+            ApiException: if unable to communicate with the Feeds Service.
+        """
+        if configuration is None:
+            configuration = core.JupyterHttpConfiguration()
```

Comment:
Make the client work with any supported HTTP Config and not just Jupyter executions on SystemLink
```suggestion
            configuration = core.HttpConfigurationManager.get_configuration()
```